app/controllers/application_controller.rb

```diff
@@ -1611,36 +1735,34 @@ def default_thumbnail_size
   end
   helper_method :default_thumbnail_size
 
-  # Set the default thumbnail size, either for the current user if logged in,
-  # or for the current session.
-  def set_default_thumbnail_size(val)
-    if @user
-      if @user.thumbnail_size != val
-        @user.thumbnail_size = val
-        @user.save_without_our_callbacks
-      end
+  def default_thumbnail_size=(val)
```

Comment:
Does thumbnail size have any real effect? Is there a UI for changing it? 
If the answers are "no", maybe the code should be ripped out (or at least rip out anything that calls `default_thumbnail` or `default_thumbnail=`.